src/main/java/io/xcherry/core/persistence/schema/local_attribute/LocalAttributeKeySchema.java

```diff
@@ -0,0 +1,48 @@
+package io.xcherry.core.persistence.schema.local_attribute;
+
+import io.xcherry.core.command.CommandResults;
+import io.xcherry.core.communication.Communication;
+import io.xcherry.core.context.Context;
+import io.xcherry.core.persistence.Persistence;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+public class LocalAttributeKeySchema {
```

Comment:
I feel like `LocalAttributeDef` is shorter and clear enough just like in iwf Java SDK

Reply:
Here is to be consistent with existing app database names. We are using xxxSchema.